lib/rubocop/cop/packaging/gemspec_git.rb

```diff
@@ -1,43 +1,57 @@
 # frozen_string_literal: true
 
+# Rubocop module
 module RuboCop
+  # Cop module
   module Cop
+    # Packaging module
     module Packaging
```

Comment:
Personally, this looks like an overkill to me and seems pretty needless.
But if I remove these comments, the documentation drops down to 54.17%:
```bash
➜  rubocop-packaging git:(master) rake generate_cops_documentation
Files:           1
Modules:         3 (    3 undocumented)
Classes:         1 (    0 undocumented)
Constants:       1 (    0 undocumented)
Attributes:      0 (    0 undocumented)
Methods:         2 (    0 undocumented)
 57.14% documented
```

I am fine with whatever you think is appropriate. So let me know what you think about this.

Reply:
ISTR that you could mark some elements with a `:nodoc:` to say explicitly that they don't need documentation

Reply:
Perfect, this worked!
So `:nodoc: it is \o/